doc/src/manual/networking-and-streams.md

```diff
@@ -312,3 +312,41 @@ resolution:
 julia> getaddrinfo("google.com")
 ip"74.125.226.225"
 ```
+
+## Asynchronous I/O
+
+All I/O operations exposed by [`Base.read`](@ref) and [`Base.write`](@ref) can be performed
+asynchronously through the use of [coroutines](@ref man-tasks). You can create a new coroutine to
+read from or write to a stream using the [`@async`](@ref) macro:
```

Comment:
Is this true for `IOStream` etc.?  I'm not a C programmer, but, naively reading `ios.c`, it seems to call the normal blocking `read` and `write`.  Also, how about the functions using `unused_uv_loop_arg`?